src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -416,6 +416,7 @@
     ),
     AndKeywordSegment=StringParser("and", KeywordSegment, type="binary_operator"),
     OrKeywordSegment=StringParser("or", KeywordSegment, type="binary_operator"),
+    NotKeywordSegment=StringParser("not", KeywordSegment, type="keyword"),
```

Comment:
Could we capitalise these keywords for consistency? Appreciate you only added one of them and followed convention but still, let's correct:

```suggestion
    AndKeywordSegment=StringParser("AND", KeywordSegment, type="binary_operator"),
    OrKeywordSegment=StringParser("OR", KeywordSegment, type="binary_operator"),
    NotKeywordSegment=StringParser("NOT", KeywordSegment, type="keyword"),
```

I also wonder if we should rename these Grammars? E.g. `AndGrammar`.

Reply:
I much prefer the CAPS.
You phrased this as a question, but "Grammar" seems good to me. I'll change it.